projects/step-core/src/lib/client/generated/services/DashboardsService.ts

```diff
@@ -0,0 +1,237 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+import { Injectable } from '@angular/core';
+import type { Observable } from 'rxjs';
+
+import type { AsyncTaskStatusTableBulkOperationReport } from '../models/AsyncTaskStatusTableBulkOperationReport';
+import type { DashboardView } from '../models/DashboardView';
+import type { History } from '../models/History';
+import type { TableBulkOperationRequest } from '../models/TableBulkOperationRequest';
+import type { TableRequest } from '../models/TableRequest';
+import type { TableResponseDashboardView } from '../models/TableResponseDashboardView';
+
+import { BaseHttpRequest } from '../core/BaseHttpRequest';
+
+@Injectable({ providedIn: 'root' })
+export class DashboardsService {
+  constructor(public readonly httpRequest: BaseHttpRequest) {}
+
+  /**
+   * Deletes the entities according to the provided parameters
+   * @param requestBody
+   * @returns AsyncTaskStatusTableBulkOperationReport default response
+   * @throws ApiError
+   */
+  public deleteEntityS5(requestBody?: TableBulkOperationRequest): Observable<AsyncTaskStatusTableBulkOperationReport> {
```

Comment:
There is something wrong with the endpoint, probably no name was provided in the swagger definiton (compare to other endpoints, we shouldn't have numbers here)